Kentico.Kontent.Management.Tests/Unit/Modules/UrlBuilder/AssetRenditionTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using FluentAssertions;
+using Kentico.Kontent.Management.Models.AssetRenditions;
+using Kentico.Kontent.Management.Models.Shared;
+using System;
+using Xunit;
+
+namespace Kentico.Kontent.Management.Tests.Unit.Modules.UrlBuilder
+{
+    public partial class EndpointUrlBuilderTests
+    {
+        [Fact]
+        public void BuildAssetRenditionsUrlFromAssetId_WithGivenAssetId_ReturnsExpectedUrl()
+        {
+            var assetId = Guid.NewGuid();
+            var expectedResult = $"{ENDPOINT}/projects/{PROJECT_ID}/assets/{assetId}/renditions";
+            var actualResult = _builder.BuildAssetRenditionsUrl(Reference.ById(assetId));
+
+            Assert.Equal(expectedResult, actualResult);
+        }
+
+        [Fact]
+        public void BuildAssetRenditionsUrlFromAssetExternalId_WithGivenAssetId_ReturnsExpectedUrl()
+        {
+            var assetExternalId = "which-brewing-fits-you";
+            var expectedResult = $"{ENDPOINT}/projects/{PROJECT_ID}/assets/external-id/{assetExternalId}/renditions";
+            var actualResult = _builder.BuildAssetRenditionsUrl(Reference.ByExternalId(assetExternalId));
+
+            Assert.Equal(expectedResult, actualResult);
+        }
+
+        [Fact]
+        public void BuildAssetRenditionsUrlFromAssetIdAndRenditionId_WithGivenAssetAndRenditionIds_ReturnsExpectedUrl()
+        {
+            var assetId = Guid.NewGuid();
+            var renditionId = Guid.NewGuid();
+            var expectedResult = $"{ENDPOINT}/projects/{PROJECT_ID}/assets/{assetId}/renditions/{renditionId}";
+            var actualResult = _builder.BuildAssetRenditionsUrl(new AssetRenditionIdentifier(Reference.ById(assetId), Reference.ById(renditionId)));
+
+            Assert.Equal(expectedResult, actualResult);
+        }
+
+        [Fact]
+        public void BuildAssetRenditionsUrlFromAssetIdAndRenditionExternalId_WithGivenAssetAndRenditionIds_ReturnsExpectedUrl()
```

Comment:
This test has in its name _WithGivenAssetAndRenditionIds_, but it actually uses renditionExternalId. Is this correct?

Reply:
Will be handled in a separate issue.